### ISSUE TITLE
Error: EEXIST using fs-extra's fs.copy to copy a directory on Windows

### ISSUE THREAD

**Issue by Kerrick** [jprichardson/node-fs-extra#89]
I have the following file structure:

```
- A
--- one.txt
--- two.txt
- B
--- three.txt
--- four.txt
```

On OS X, I end up with a `C` directory containing all four `.txt` files, as I'd expect. On Windows, when I try to `fs.copy('A/', 'C');` and then `fs.copy('B/', 'C')`, the second one fails with the following error:

```
[copy ERROR] Error: EEXIST, mkdir 'C:\testing\C'
```

**Comment by jprichardson** [comment]
Thanks for reporting. I'm not sure why I didn't follow up immediately awhile back, sorry about that. I've started working on this https://github.com/jprichardson/node-fs-extra/blob/master/test/copy/async/copy-gh-89.test.js But while writing the test, I realized that it's not entirely clear how `fs-extra` should handle this issue today. They should definitely be consistent, that's for sure.

`fs-extra` doesn't **yet** properly handle copy commands liked you'd expect with `cp` (posix) or `[x]copy` (windows). This is definitely coming. 

The problem that I have is that `fs.copy('A/', 'C')` is different than `fs.copy('A', 'C')`. Like I said, long-term, `fs-extra` will support this. Right now, for the sake of simplicity, I'm hesitant too.

I'll need to think more about this. Would love to hear any thoughts that you have.
